Dockerfile

```diff
@@ -0,0 +1,17 @@
+FROM node:14 AS build
+
+WORKDIR /app
+
+COPY package*.json ./
+RUN npm ci --only=production
+
+COPY . .
+
+FROM gcr.io/distroless/nodejs:14
```

Comment:
I noticed that you are specifying node 14+ here and I'm assuming it's because it's the latest stable version. 

I would prefer that the version align with that specified in the [CI](https://github.com/mergeability/mergeable/blob/master/.circleci/config.yml) since that's what we run our test suite against. Thoughts?